nce/NCE_MP_Pytorch/trainers/trecqa_trainer.py

```diff
@@ -0,0 +1,7 @@
+from nce.NCE_MP_Pytorch.trainers.qa_trainer import QATrainer
+
+
+class TRECQATrainer_NCE(QATrainer):
```

Comment:
Please read Python convention on how to name classes here: https://www.python.org/dev/peps/pep-0008/#class-names and update names to adhere to this convention in all places in the PR.

Please do not use snake case (no underscore), change this to `TRECQATrainerNCE`.

Reply:
Great thanks for your suggestion!!!